package.json

```diff
@@ -298,10 +298,9 @@
     "@emotion/core": "^10.0.16",
     "@loadable/babel-plugin": "^5.7.1",
     "@loadable/webpack-plugin": "^5.2.1",
-    "@storybook/addon-info": "5.3.21",
```

Comment:
addon-info is deprecated for quite some time.. We were using it for displaying props of some components but it didn't always work well (and had issues with Flow in the past). The new addon-docs seems better.